### PR TITLE
Update datamash to latest stable version 1.1.0

### PR DESCRIPTION
**Pull request by dmaticzka**
* [x] I have read the guidelines above.
* [ ] This PR adds a new recipe.
* [x] This PR updates an existing recipe.
* [ ] This PR does something else (explain below).

### CODE REVIEW

File: recipes/datamash/1.1.0/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+  summary: 'GNU Datamash is a command-line program which performs basic numeric,textual
+    and statistical operations on input textual data files.'
+build:
+  number: 0
+  skip: True # [osx]
+package:
+  name: datamash
+  version: 1.1.0
+requirements:
+  build: []
```
Comment (bgruening) [inline recipes/datamash/1.1.0/meta.yaml]: Please depend on `gcc # [linux]` and `llvm # [osx]`.

File: recipes/datamash/1.1.0/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,21 @@
+about:
+  home: http://www.gnu.org/software/datamash
+  license: GPL3
+  summary: 'GNU Datamash is a command-line program which performs basic numeric,textual
+    and statistical operations on input textual data files.'
+build:
+  number: 0
+  skip: True # [osx]
```
Comment (bgruening) [inline recipes/datamash/1.1.0/meta.yaml]: Is this still needed?

**Comment by dmaticzka** [comment]
Added the gcc and llvm dependencies for both datamash versions. Tests run fine for osx.

**Comment by bgruening** [comment]
Awesome thanks!

**Pull request merged**